www-src/app/application.coffee

```diff
@@ -60,16 +61,30 @@ module.exports =
                 DeviceStatus.initialize()
 
                 if config.remote
-                    unless @replicator.config.has('checkpointed')
-                        log.info 'Launch first replication again.'
-                        app.router.navigate 'first-sync', trigger: true
-                    else
-                        app.regularStart()
+                    app.replicator.checkPlatformVersions (err) =>
+                        if err
+                            log.error err
+                            alert err.message or err
+                            return navigator.app.exitApp()
+
+                        unless config.hasPermissions()
+                            app.router.navigate 'permissions', trigger: true
+
+                        else
+                            # TODO : try to move to regular start.
+                            unless @replicator.config.has('checkpointed')
+                                log.info 'Launch first replication again.'
+                                app.router.navigate 'first-sync', trigger: true
+                            else
+                                app.regularStart()
```

Comment:
@jacquarg I prefer to avoid

```coffee
unless cond
  # [1]
else
  # [2]
```

and use instead

```coffee
if cond
  # [2]
else
  # [1]
```

(It's just a personal preference)